Formula/cfitsio.rb

```diff
@@ -39,4 +40,18 @@ def install
       end
     end
   end
+
+  test do
+    cp Dir["#{pkgshare}/testprog/testprog*"], testpath
+    flags = (ENV.cflags || "").split + (ENV.cppflags || "").split + (ENV.ldflags || "").split
```

Comment:
I don't think it is necessary; I think it was some sort of early "handle superenv if we ever port tests to that" or something. I don't think there's any need for it, really.